server/email/email-controller.js

```diff
@@ -0,0 +1,55 @@
+var mandrill = require('./email-service');
```

Comment:
i think the mandrill variable should be renamed to something like 'emailService' since it is supposed to be an abstraction above mandrill. thoughts?